aws-modules/aws-lambda-modules/lambda-function/src/main/java/com/baeldung/lambda/LambdaStreamHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.lambda;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class LambdaStreamHandler implements RequestStreamHandler {
+
+    @Override
+    public void handleRequest(InputStream input, OutputStream output, Context context) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        Request request = mapper.readValue(input, Request.class);
+
+        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(output));
+        writer.write("Hello " + request.name() + ", Baeldung has great Java content for you!");
+        writer.flush();
```

Comment:
Can we use a try-with-resources block here, or is there some specific reason not to?

Reply:
Yes, we should use try-with-resources here, review incorporated.